packages/hackney.exs

```diff
@@ -19,7 +19,7 @@ defmodule Hackney.Mixfile do
   defp deps do
     [
       {:idna, "1.0.2"},
-      {:ssl_verify_hostname, "1.0.1"},
+      {:ssl_verify_hostname, "1.0.3"},
```

Comment:
I don't think we need this strict version requirement. Relaxed requirements are usually preferred over strict ones if you are not sure of the exact requirements (which is usually the case for community packages). I will merge this and fix manually so we don't have more back and forth :)